assets/lib/commands/out.rb

```diff
@@ -92,6 +92,8 @@ def output
     private
 
     def whitelist(context:)
+      return nil unless context
```

Comment:
The `context` is a required field. The `nil` check makes me think something is calling this incorrectly. Do you have a test that reproduces the issue?